docs/governance/code-governance.rst

```diff
@@ -0,0 +1,237 @@
+
+Tier 2
+======
+Minor features or enhancements which do not significantly change any part of Mautic.
+
+More complex bug fixes are also probably going to be in Tier 2, as well as PRs that are related to external services (for example, Salesforce/HubSpot) for which accounts need to be created in order to test.
+
+Examples (Tier 2)
+------------------
+
+:xref:`github_pr_6090`- new feature, fairly simple and doesn’t significantly change any major part of Mautic
+
+:xref:`github_pr_7432`- a lot of new code which extends an existing feature, enhancing reports with a scheduling option
+
+
+Requirements for merging a Tier 2 PR
+--------------------------------------
+
+* Thorough testing to ensure that it does what it’s expected to do and does not break anything else
+* Full unit testing coverage
+* Full documentation support if it’s a feature change/addition
+* Code review from 1 core team member
+* Full testing and approval by at least one community member (could be the person doing the code review, but should not be from the same company/organisation as the person submitting the PR as mentioned above)
+
+Tier 3
+======
+Major changes touching multiple parts of Mautic, completely changing the way some aspect of the code works, or large amounts of code changed.
+
+Examples (Tier 3)
+------------------
+
+:xref:`github_pr_6875`- complex code changes across multiple areas, with significant impact
+
+:xref:`github_pr_6584`- not too complex but a large visual change in the UI with the potential to cause significant disruption to the end user
+
+
+Requirements for Merging a Tier 3 PR    
+-------------------------------------
+* Thorough testing to ensure that it does what it’s expected to do and does not break anything else
+* Thorough testing across the whole product
+* Full unit/functional test coverage
+* Full documentation support
+* Code review from 1-2 core team members
+* Full testing and approval by at least two community members (could be the people doing the review, but should not be from the same company/organisation as the person submitting the PR as mentioned above)
+* 2 week community review period - after which Product Team decides via a vote in the next available team meeting whether to merge (assuming code review and all other requirements are in place to permit merging)
+
+.. note::
+    Tier 3 PRs which relate to new features that impact users of Mautic **should be linked to in an individual forum thread** at :xref:`Ideas and Feature Requests` and discussed on the forums. This gives the non-technical user base an opportunity to be involved in the discussions on new features. 
+
+
+If a thread doesn’t exist prior to the PR being submitted (for example, a feature created without the community suggesting it), it should be created during triage when a Tier 3 & Feature label is added.
```

Comment:
```suggestion
If a thread doesn’t exist prior to the PR being submitted - for example, a feature created without the community suggesting it - it should be created during triage when a Tier 3 & Feature label is added.
```

This will also need converting to active voice. Vale hasn't picked it up yet.